pkg/cmd/cli/cmd/login/logout.go

```diff
@@ -123,25 +125,40 @@ func (o LogoutOptions) RunLogout() error {
 		return err
 	}
 
-	if err := client.OAuthAccessTokens().Delete(token); err != nil {
-		return err
+	errs := []error{}
+	serverErr := client.OAuthAccessTokens().Delete(token)
+	errs = append(errs, serverErr)
+
+	if configErr := deleteTokenFromConfig(*o.StartingKubeConfig, o.PathOptions, token); configErr != nil {
+		errs = append(errs, configErr)
+	} else if serverErr != nil {
+		// if token was not able to be deleted using api but was successfully removed
+		// from client config, log brief explanation that token was still removed locally
+		glog.V(1).Infof("%s\n\n", "An error ocurred deleting the token on the server. The token has been removed from your local configuration.")
```

Comment:
You're doing a `glog.V(1).Infof` with the flattened error below, so you'll always get a log about the delete failure. You should just do this here:

```go
if configErr := deleteTokenFromConfig(*o.StartingKubeConfig, o.PathOptions, token); configErr == nil {
    glog.V(1).Info("Removed token from your local configuration.\n\n")
}
errs = append(errs, configErr)
```